dynamicArray.cpp

```diff
@@ -1,20 +1,66 @@
-// dynamicArray.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
+//Created by Oheruk Taras
+/*
+    TODO:
+    1.���������� � size (��� ����� �������� �� ���)
+    2.
+*/
 
 #include <iostream>
+#include <time.h>
+#include <string>
 
-int main()
-{
-    std::cout << "Hello World!\n";
-}
-
-// Run program: Ctrl + F5 or Debug > Start Without Debugging menu
-// Debug program: F5 or Debug > Start Debugging menu
-
-// Tips for Getting Started: 
-//   1. Use the Solution Explorer window to add/manage files
-//   2. Use the Team Explorer window to connect to source control
-//   3. Use the Output window to see build output and other messages
-//   4. Use the Error List window to view errors
-//   5. Go to Project > Add New Item to create new code files, or Project > Add Existing Item to add existing code files to the project
-//   6. In the future, to open this project again, go to File > Open > Project and select the .sln file
+using namespace std;
+
+class List {
+private:
+    int* array;
+    int size;
+    int capacity;
+
+public:
+    List() {
+        size = 0;
+        capacity = 5;
+        array = new int[capacity];
+    }
+
+    void add(int index, int value) {
```

Comment:
the API for dynamic vector should have following functions:
* push_back - adds element to the end of array.
* push_front - adds element to the start of array.
* pop_back - removes element from the end of array.
* pop_front - removes element from the start of array.
* clear - clears dynamic array.
* remove(int index) removes element from array (size should become size-1)